controllers/calendar.ts

```diff
@@ -0,0 +1,37 @@
+import { Request, Response } from "express";
+import { google } from "googleapis";
+import config from "config";
+
+const gcalClientId = config.get("calendarAccessToken.gcal.clientId");
+const gcalClientSecret = config.get("calendarAccessToken.gcal.clientSecret");
+const calApiUrl = config.get("services.calendarApi.baseUrl");
+
+const oauth2Client = new google.auth.OAuth2(
```

Comment:
Can we define this inside `googleConnectHandler()` as this isnt used anywhere else?

Reply:
This will be needed in phase 3 of the ticket #40